app/javascript/controllers/slim_select_controller.js

```diff
@@ -23,6 +23,28 @@ export default class extends Controller {
           }
           this.element.dispatchEvent(new Event("blur"))
         },
+        beforeChange: (newOpts, oldOpts) => {
```

Comment:
Here's the UX magic.  Since I combined all the "weeks of the month" options into 1 select ... some of those are exclusive.

So when you select "Every two weeks", this deselects anything else.

Reply:
is the returned boolean what deselects/clears the existing values? so, `beforeChange` (before the value changes) we want to set the new value and clear whatever has been selected? I'm not familiar with the inner workings of the SS controller.

Reply:
Yep - [returning false from beforeChange](https://slimselectjs.com/events#beforeChange) prevents the change from happening.  Which is okay in those 2 special cases, since I'm manually changing `setSelected(...)` myself.